DTS-STN/dts-rmp
# 249 updating seeder values

Updated Jordan's seeder to match the keys used on the Select tags of the contact form

## Test Instructions

1. Usual steps
1. npm run mongodb
1. npm run seed
1. on edit mode of a contact the Drop-down boxes should have a value 

## Checklist
